components/AvalancheForecastZoneMap.tsx

```diff
@@ -273,7 +276,7 @@ const AvalancheForecastZoneCard: React.FunctionComponent<{
         });
       }}>
       <VStack borderRadius={8} bg="white" width={width * CARD_WIDTH} marginX={CARD_MARGIN * width}>
-        <View height="8px" width="100%" bg={dangerColor.string()} borderTopRadius={8} pb={0} />
+        <View height={8} width="100%" bg={dangerColor.string()} borderTopLeftRadius={8} borderTopRightRadius={8} pb={0} />
```

Comment:
I didn't bother adding a facility for having one pseudo prop (`borderTopRadius`) set two props in the final style. If it becomes common I'll consider it, but I don't see the need yet.